### DISCUSSION TITLE
File explorer integration

### DISCUSSION THREAD

**Discussion by devistar** [Flow-Launcher/Flow.Launcher#1031] [Ideas]
Hello,
I used Listary for a long time and I liked it but this software has not been updated for a long time. Then I looked for alternatives and I find that Flow Launcher is very interesting.
However, there is a Listary feature that was great not implemented in Flow Launcher. It's integration with the file explorer, whether Windows Explorer or XYPlorer. It is indeed possible when you are in a directory to carry out a search. When opening a file, you can search or use a hotkey to go to the opened directory in the file explorer.
It would be great if you could add this feature to Flow Launcher. :-)

![image](https://user-images.githubusercontent.com/11228594/154798179-c88ff236-9138-47b4-b9c8-eea45049f7d1.png)

![image](https://user-images.githubusercontent.com/11228594/154798131-bd6ae049-5ed2-4964-b6d3-117f56745e3a.png)

**Comment by taooceros** [comment]
This is a hard feature to implement because I (and other contributors) have limited knowledge about the dialog api for navigating and accessing path. Therefore, we still have no idea how to implement this in a fluent way as listary has done.
